main.sh

```diff
@@ -355,6 +356,11 @@ register_binfmt() {
 
 setup_linux_host() {
     apt_packages=()
+    while read -rd,; do
+        if [[ -n "${REPLY}" ]]; then
+            apt_packages+=("${REPLY}")
```

Comment:
To install a package for the target different from the host, the `<package>:<dpkg-arch>` syntax needs to be used.

Reply:
Ah yea my mistake, in my testing i misread `amd64` as `arm64`. I'll look into adding the `dpkg-arch` to the package.